rollup.config.export-dev.mjs

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import clear from 'rollup-plugin-clear';
+import copy from 'rollup-plugin-copy';
+import esbuild from 'rollup-plugin-esbuild';
+import resolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import { replaceConfig } from './build_common.mjs';
+
+const outDir = 'dist_dev';
+const entryFileNamesPattern = '[name].ts';
+const replacePluginInstance = replace(replaceConfig('/'));
+
+export default [
+  {
+    input: 'src/index.ts',
+    output: {
+      dir: outDir,
+      sourcemap: true,
+      entryFileNames: entryFileNamesPattern,
+    },
+    plugins: [
+      clear({ targets: [outDir] }),
+      replacePluginInstance,
+      esbuild({}),
```

Comment:
Here and below (:60): why don't we need `{ ts: true }`?

Reply:
That doesn't seem to be an option: https://www.npmjs.com/package/rollup-plugin-esbuild

(note that here we're using `rollup-plugin-esbuild`, not `@web/dev-server-esbuild` - I guess `rollup-plugin-esbuild` has typescript support unconditionally enabled).

Reply:
Curious, quite odd for those APIs to be different. SG since it works.